readthedocs/doc_builder/templates/doc_builder/readthedocs-data.html

```diff
@@ -0,0 +1,6 @@
+{{ data|json_script:"READTHEDOCS_DATA" }}
```

Comment:
If this file is just JSON, and we're consuming it by a request, does it need to be HTML? I'll note above what I'm a bit confused by

Reply:
No, it does not need to be HTML. I'm doing it like this here just because I quickly found that `json_script` was the easiest way to generate a JSON object. However, that template filter wraps the object with a `<script>` HTML tag.

The only thing I wanted to show with this file was the concept of "how to consume data generated at build time from the frontend".